redbadger/continous-compliance
# Gathering GitHub issues evidence into compliance folder

closes #18 

## How to test

1. Go to `count-dracula` code and pull recent remote branches
```sh
git fetch
```
2. Go to branch `test-continous-compliance-v07`
```sh
git checkout test-continous-compliance-v07
```
3. Do an empty commit and push to remote
```sh
git commit --allow-empty -m "Trigger continous-compliance CI/CD"
git push
```
4. Go to count dracula [actions](https://github.com/redbadger/count-dracula/actions)
5. Wait until `continous compliance` step on **continous-compliance** job finished and open the job.
6. You should see a message like this
```txt
Saving GitHub evidence on compliance/github/info.json
Gathering information about PR #36
Gathering information about commits associated with PR #36 📝
Saving GitHub evidence on compliance/github/info.json
```
7. Click the URL on the last line and should trigger a file download
8. Check download file for a folder called `github`
8. 🚀 
![](https://media.giphy.com/media/dxn6fRlTIShoeBr69N/giphy.gif)
